module/zfs/spa_misc.c

```diff
@@ -376,7 +376,8 @@ int spa_asize_inflation = 24;
  * See also the comments in zfs_space_check_t.
  */
 int spa_slop_shift = 5;
-uint64_t spa_min_slop = 128 * 1024 * 1024;
+uint64_t spa_min_slop = 128ULL * 1024 * 1024;
+uint64_t spa_max_slop = 128ULL * 1024 * 1024 * 1024;
```

Comment:
You'll want to add the module parameter declaration (and manpage change) for this.

Reply:
Yea, I wasn't sure if we would actually want to expose this to an administrator, so I kept it conservative and hadn't done that yet. If you think this is something worth exposing, then I can do that.

By exposing it via a parameter, I felt that it might enable a naive administrator to inadvertently shoot themselves in the foot, causing their pool to be unrecoverable (e.g. due to ENOSPC issues, resulting from setting it too low), so I thought keeping it "hidden" and only able to be changed in the code might be better.

Reply:
That makes sense.   Could you change the PR description to make it clear that this isn't a module parameter ("tunable")?

Reply:
Done; I changed it to use "variable" instead of "tunable". Hope that more accurately reflects my intent.